models/yang/sonic/sonic-mgmt-port.yang

```diff
@@ -0,0 +1,75 @@
+module sonic-mgmt-port {
+    namespace "http://github.com/Azure/sonic-mgmt-port";
+    prefix smgmtprt;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import sonic-common {
+        prefix scommon;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC Management Interface";
+
+    revision 2019-09-17 {
+        description
+            "Initial revision.";
+    }
+
+
+    container sonic-mgmt-port {
+
+        container MGMT_PORT {
+
+            list MGMT_PORT_LIST {
+
+                key "ifname";
+
+                leaf ifname {
+                    type string {
+                        pattern "eth([1-3][0-9]{3}|[1-9][0-9]{2}|[1-9][0-9]|[0-9])"{
+                            error-message "Invalid interface name";
+                            error-app-tag interface-name-invalid;
+                        }
+                    }
+                }
+
+                leaf speed {
+                    type uint64;
```

Comment:
Any constraint ?

Reply:
What is the resolution here ? Do we have any constraint ?

Reply:
@dutta-partha - Based on discussion we need to conclude on platform deviations etc. How are you handling this in SONiC ?

Reply:
Platform specific constraint through deviation file has been added and should be available in 'master' branch once sync from broadcom_sonic branch happens.

Reply:
@bhavini-gada Support for platform constraint through deviation file is available in broadcom_sonic_share branch.

Reply:
Allowed speeds for Management interface 10, 100 and 1000 MBPS, this comment is addressed on Friday. Can you please take a look and if you dont have any comments. Please mark as resolved. For Physical interfaces we might need platform deviations.